### ISSUE TITLE
Homebr3w hangs when system fails to validate certificate for https links

### ISSUE THREAD

**Issue by 13xforever** [Wolvan/Homebr3w#32]
Repro: try to download freeShop

**Comment by Wolvan** [comment]
@ksanislo shouldn't https certificate check be disabled?

**Comment by ksanislo** [comment]
Yeah, the cert check is actually disabled... I've mentioned this to him already, but the author has a problem with the cert he's using on the freeshop.pw site that causes a failure even when the official 3DS check is turned off. There's no solution from this end, as long as he keeps letting release links get posted that have the bad certificate, it will continue to show up from time to time.
